cg/apps/tb/api.py

```diff
@@ -178,3 +178,12 @@ def get_summaries(self, order_ids: list[int]) -> list[AnalysisSummary]:
         response = self.query_trailblazer(command=endpoint, request_body={}, method=APIMethods.GET)
         response_data = SummariesResponse.model_validate(response)
         return response_data.summaries
+
+    def get_analyses_to_deliver(self, order_id: int) -> list[TrailblazerAnalysis]:
```

Comment:
Design: this method is not an "atomic" operation and should be in a service instead of the web API client. Out of scope though

Reply:
Another PR but I like the idea